src/panels/developer-tools/statistics/dialog-statistics-adjust-sum.ts

```diff
@@ -63,6 +63,11 @@ export class DialogStatisticsFixUnsupportedUnitMetadata extends LitElement {
         @closed=${this.closeDialog}
         heading="Adjust sum for a specific time."
```

Comment:
```suggestion
        heading="Adjust sum statistics from a specified time and onwards."
```

Also, this should move to translations.

Reply:
None of the stats dev tools have translations 🤷 

I don't think that we need to add "onwards", that is kinda like "duh". Under the hood we apply it onwards but visually for the user nothing changes.